### PR TITLE
Fixes partner link on Fair booth header

### PR DESCRIPTION
**Pull request by l2succes**
Fixes: https://artsyproduct.atlassian.net/browse/LD-247

### CODE REVIEW

File: src/lib/Scenes/Fair/Components/SearchLink.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export class SearchLink extends React.Component<Props> {
 
   render() {
     return (
-      <TouchableWithoutFeedback onPress={() => this.handlePress}>
+      <TouchableWithoutFeedback onPress={() => this.handlePress()}>
```
Comment (l2succes) [inline src/lib/Scenes/Fair/Components/SearchLink.tsx]: Just noticed the search bar on Fair was broken and bundled that fix there as it's very small

**Comment by l2succes** [comment]
@ashleyjelks Gonna merge this one as it's #trivial

File: src/lib/Scenes/Show/Screens/Detail.tsx [hunk 2]
```diff
@@ -100,10 +100,12 @@ export class Detail extends React.Component<Props, State> {
       },
     })
 
-    sections.push({
-      type: "shows",
-      data: show,
-    })
+    if (show.location) {
```
Comment (ashkan18) [inline src/lib/Scenes/Show/Screens/Detail.tsx]: should we use `show.has_location`? the logic behind `has_location` is slightly different and thats what defines if the show is online exclusive or not. What do you think? it's already exposed in MP for `Show`

Comment (l2succes) [reply src/lib/Scenes/Show/Screens/Detail.tsx]: Ahhh, just saw this. Yeah that makes sense. I'll make a separate PR to use `has_location` instead

**Pull request merged**